### ISSUE TITLE
Get_fingerprint() implementation

### ISSUE THREAD

**Issue by HynekPetrak** [ronf/asyncssh#182]
Hi Ron,

can you please add a method `get_fingerprint(self, type)` to the SSHKey class. The method would return fingerprint like a call to `ssh-keygen -l -f key`?

Input parameter `type` would be either "md5" or "sha256"

Then:
```
a = self.get_algorithm()
pubk = struct.pack(">I", len(a)) +  a.encode('ascii') + self.encode_ssh_public()
```

then fingerptint for `type == "md5"` is:
```
hd_md5 = hashlib.md5(pubk).hexdigest()
fingerprint_md5 = "MD5:" + ':'.join(hd[i:i+2] for i in range(0, len(hd), 2))
```

and for `type == "sha256"`:
```
fingerprint_sha256 = "SHA256:" + base64.b64encode(hashlib.sha256(pubk).digest()).decode('ascii').strip("=")
```

**Comment by ronf** [comment]
This is now available in the "develop" branch in commit 86ffd46, and will be added in the next release. It supports md5, sha1, sha256, sha384, and sha512 options for the hash. All the SHA variants are encoded as base64, but the md5 variant is encoded as hex pair like you show here, matching what ssh-keygen in OpenSSH outputs.

**Comment by ronf** [comment]
The support described here is now released in AsyncSSH 1.15.1.
